WalletWasabi/Services/SingleInstanceChecker.cs

```diff
@@ -138,7 +138,7 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 				while (!stoppingToken.IsCancellationRequested)
 				{
 					// In case of cancellation, listener.Stop will cause AcceptTcpClientAsync to throw, thus canceling it.
-					using var client = await listener.AcceptTcpClientAsync().WithAwaitCancellationAsync(stoppingToken).ConfigureAwait(false);
```

Comment:
.NET 6 has added an overload for `AcceptTcpClientAsync` with cancellation token as its parameter.

Reply:
What's the plan? Do you want to change it in this PR?

Reply:
It seems worth it to do the change as the thrown exception should still be `OperationCanceledException` (https://github.com/dotnet/runtime/pull/53340).